src/BotService/Infrastructure/Core/CallHandler.cs

```diff
@@ -661,6 +694,9 @@ private async void CallOnUpdated(ICall sender, ResourceEventArgs<Call> e)
                 await _mediatorService.SetCallAsEstablishedAsync(callId, Call.Id);
 
                 _logger.LogInformation("[CallHandler] Call Established {callId}", Call.ScenarioId.ToString());
+
+                // Start slate image injection
```

Comment:
**question(non-blocking)**: Is this comment necessary? It says the same as the name of the method. If we are gonna put a comment, maybe we can describe the reason we are doing that injection.

Reply:
Ok